lib/api/sentry_api.dart

```diff
@@ -57,7 +57,7 @@ class SentryApi {
   Future<Set<String>>projectIdsWithSessions(String organizationSlug) async {
     final queryParameters = <String, dynamic>{ /*String|Iterable<String>*/
       'project': '-1',
-      'interval': '90d',
+      'interval': '1d',
```

Comment:
@denrase Do you need to do some reducing on the client-side as now you won't get one bucket from the backend but rather 90?

Reply:
I don't think we need to do anything else, as we are only interested in the project ids from this call and not the interval data.

Reply:
ah, cool 👍